### PR TITLE
🌱 Move CecileRobertMichon to emeritus

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**What this PR does / why we need it**: Moving myself to emeritus. After 6+ years at Microsoft and the last 3+ years spent working with the Cluster API community, I've decided to try something new which means I'll have less time to dedicate to this project going forward. I am removing myself as an approver to make room for future approvers. I've learned so much from the folks in this community and am proud to see how far the project has come. I will still be around in the k8s slack, feel free to reach out! 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -14,12 +14,11 @@ aliases:
   # active folks who can be contacted to perform admin-related
   # tasks on the repo, or otherwise approve any PRS.
   cluster-api-admins:
-  - CecileRobertMichon
+  - fabriziopandini
```
Comment (CecileRobertMichon) [inline OWNERS_ALIASES:17]: Moving @fabriziopandini to admins for redundancy 

cc @vincepri

**Comment by sbueringer** [comment]
/lgtm

sort of 😅 

As already written elsewhere. Thank you for everything you did for the community. I was always great working with you and I hope to see you around :)

**Review by killianmuldoon — COMMENTED** [review]
/area community-meeting

To match #9997

**Comment by killianmuldoon** [comment]
Thanks for everything Cecile!

/lgtm

**Comment by enxebre** [comment]
thanks for all the work @CecileRobertMichon

**Review by vincepri — APPROVED** [review]
/approve
I DO NOT 
❤️

**Comment by dims** [comment]
😢

**Comment by elmiko** [comment]
thanks for all the advice and guidance @CecileRobertMichon , it's a true pleasure having you as a colleague in this community.

**Pull request merged**